src/containers/BooksForm/BooksForm.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { createBook } from '../../actions';
+
+class BooksForm extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      title: '',
+      category: 'Action',
+    };
+  }
```

Comment:
- `BookForm` component should be placed in `src/containers` directory, not `src/containers/BooksForm`